docs/docs/building.md

```diff
@@ -37,26 +37,73 @@ Instead of installing the dependencies system-wide, you can also opt to use [`vc
 
 The following commands should work on all platforms:
 
+### Clone sources
+
 ```bash
 git clone --recurse-submodules https://github.com/valhalla/valhalla
 cd valhalla
-
+# Clone vcpkg for dependencies:
 git clone https://github.com/microsoft/vcpkg && git -C vcpkg checkout <some-tag>
+```
+### Bootstrap `vcpkg`
+
+```bash
+# linux / macOS
```

Comment:
just thinking out loud, no need for action. but would be cool to have the `mkdocs` extension which can generate tabs so we have only one code box in these situations with tabs ["Linux/macOS", "Windows"]. at least I assume that extension exists

Reply:
I've seen documentations that does that, even defaulting to the OS the user is on. I can poke around for something like that for `mkdocs`.

Reply:
I actually found it when reviewing #5296 : https://squidfunk.github.io/mkdocs-material/reference/content-tabs/. If you have the will power for it, feel free to include it. Otherwise we can leave it for later:)

Reply:
I can look into it, but I'd prefer to do that in a separate PR if that's ok.

Reply:
> I actually found it when reviewing #5296 : https://squidfunk.github.io/mkdocs-material/reference/content-tabs/. If you have the will power for it, feel free to include it. Otherwise we can leave it for later:)

Is valhalla docs using Material theme?

Reply:
Quick PR for the Building page: https://github.com/valhalla/valhalla/pull/5298

There might be more places in the docs where this is useful, but that can be added incrementally as they are found.